docs/porting/third-party-deps.md

```diff
@@ -1,19 +1,69 @@
 # Analyzing your 3rd Party Dependencies
 
-TODO intro
+The first step in the porting process is to understand your third party dependencies, figure out which of them (if any), don't run on .NET Core, and develop a contingency plan for those which don't run on .NET Core.
 
 ## Prerequisites
 
-TODO prerequisites
+This article will assume you are using Windows and Visual Studio.  For your NuGet package dependencies, you may also wish to download the [Nuget Package Explorer](https://github.com/NuGetPackageExplorer/NuGetPackageExplorer).
 
 ## Analyzing NuGet Packages
 
-TODO nuget
+Analyzing NuGet packages for portability is very easy.  Because a NuGet package is itself a set of folders which contain platform-specific assemblies, all you have to do is check to see if there is a folder which contains a .NET Core assembly.
+
+Inspecting NuGet Package folders is easiest with the [NuGet Package Explorer](https://github.com/NuGetPackageExplorer/NuGetPackageExplorer).  Here's how to do it.
+
+1. Open the NuGet Package Explorer.
+2. Click "Open package from online feed".
+3. Search for the name of the package.
+4. Expand the "lib" folder on the right-hand side and look at folder names.
+
+You can also see what a package supports on [nuget.org](nuget.org) under the **Dependencies** section of the page for that package.
+
+In either case, you'll need to look for a folder or entry on [nuget.org](nuget.org) with any of the following names:
```

Comment:
Yes, that's necessary.

See: https://github.com/dotnet/core-docs/blob/master/docs/standard/library.md#pcl-compatibility